### PR TITLE
vulkaninfo: shorten format dump in console

### PR DESCRIPTION
**Pull request by krOoze**

### CODE REVIEW

File: vulkaninfo/vulkaninfo.c [hunk 1]
```diff
@@ -2008,24 +2008,172 @@ bool FormatRangeSupported(const struct FormatRange *format_range, const struct A
...
+
+                    if( propFormats->format_count > propFormats->format_reserve ){
+                        propFormats->format_reserve *= 2;
+                        propFormats->formats = realloc( propFormats->formats, sizeof(VkFormat) * propFormats->format_reserve);
+                        if (!propFormats->formats) ERR_EXIT(VK_ERROR_OUT_OF_HOST_MEMORY);
+                    }
+
+                    propFormats->formats[propFormats->format_count - 1] = fmt;
+                }
+                else {
```
Comment (jeremyk-lunarg) [inline vulkaninfo/vulkaninfo.c]: The checks will get mad at this do to improper formatting.  Make sure you run clang-format on your changes.

Comment (krOoze) [reply vulkaninfo/vulkaninfo.c]: I did, but I think something is wrong with it as it did no changes :/

Comment (jeremyk-lunarg) [reply vulkaninfo/vulkaninfo.c]: Just tried it on mine and no changes either.  Well, let's see what CI says and if it's good then who am I to argue.

Comment (krOoze) [reply vulkaninfo/vulkaninfo.c]: Google is C++ style, no? Maybe it just skips C files?

Comment (krOoze) [reply vulkaninfo/vulkaninfo.c]: Oh, there's `clang-format off` macro in the code :D

**Pull request merged**